vendor/VREffect.js

```diff
@@ -232,7 +232,7 @@ THREE.VREffect = function ( renderer, onError ) {
 	var cameraR = new THREE.PerspectiveCamera();
 	cameraR.layers.enable( 2 );
 
-	this.render = function ( scene, camera ) {
+	this.render = function ( scene, camera, renderTarget, forceClear ) {

```

Comment:
I believe so, this is the signature of WebGLRenderer.render() and since VREffect behaves similarly it should match it.